taglib/mpeg/id3v2/id3v2footer.h

```diff
@@ -70,7 +70,7 @@ namespace TagLib {
       /*!
        * Renders the footer based on the data in \a header.
        */
-      ByteVector render(const Header *header) const;
+      static ByteVector render(const Header *header);
```

Comment:
I believe @ufleisch stated that TagLib 2 should be source-compatible so this change would break that requirement. Personally, I also like having `render` as an instance method for parity with the other classes even if it would work as a static.